otx/algorithms/classification/adapters/mmcls/datasets/pipelines/transforms/augmix.py

```diff
@@ -1,6 +1,8 @@
+"""Module for defining AugMix class used for classification task."""
 # Copyright (C) 2022 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 #
+# pylint: disable=too-many-instance-attributes, too-many-locals, too-many-arguments, unused-argument, invalid-name
```

Comment:
Please disabling pylint as minimum as possible. How about setting in on necessary functions or classes?